src/combined_ice_ocean_driver.F90

```diff
@@ -332,6 +356,71 @@ subroutine direct_flux_ice_to_IOB(Time, Ice, IOB, do_thermo)
 
 end subroutine direct_flux_ice_to_IOB
 
+!> This subroutine does a direct copy of the fluxes from the ocean public type into
+!! a ocean-ice boundary type on the same grid.
+!! This is analogous to the flux_ocean_to_ice subroutine in ice_ocean_flux_exchange.F90
+!! but assumes the sea ice and ocean are on the same grid and does a direct copy as in
+!! direct_flux_ice_to_IOB above. The thermodynamic varibles are also seperated so only 
+!! the dynamics are updated.
+!! The data_override is similar to flux_ocean_to_ice_finish
+subroutine direct_flux_ocn_to_OIB(Time, Ocean, OIB, Ice, do_thermo)
+  type(time_type),    intent(in)     :: Time  !< Current time
+  type(ocean_public_type),intent(in) :: Ocean !< A derived data type to specify ocean boundary data
+  type(ocean_ice_boundary_type), intent(inout)   :: OIB !< A type containing ocean surface fields that
+                                                      !! are used to drive the sea ice
+  logical,  optional, intent(in)     :: do_thermo !< If present and false, do not update the
+                                              !! thermodynamic or tracer fluxes.
+  type(ice_data_type), &
+    intent(inout) :: Ice            !< The publicly visible ice data type in the slow part
+                                    !! of which the ocean surface information is to be stored.                                            
+  logical :: used, do_therm, do_area_weighted_flux
+
+  call cpu_clock_begin(fluxOceanIceClock)
+
+  do_therm = .true. ; if (present(do_thermo)) do_therm = do_thermo                                              
+  do_area_weighted_flux = .false. !! Need to add option to account for area weighted fluxes                           
+                                              
+  if( ASSOCIATED(OIB%u)     )OIB%u = Ocean%u_surf
```

Comment:
There should be white space after the `if` and between the closing ")" and OIB here and in the following lines in this routine, to read something like `if ( ASSOCIATED(OIB%u) ) OIB%u = Ocean%u_surf` to follow the MOM6 (and SIS2) style guide.